### DISCUSSION TITLE
Error with JMeter 5.6.2 in connection with plugin kloadgen 5.6.5

### DISCUSSION THREAD

**Discussion by xck1007** [sngular/kloadgen#406] [Q&A]
Only when I use the kloadgen plugin 5.6.5 do I see exceptions in the jmeter log file after starting jmeter 5.6.2. I haven't used any function of kloadgen until then. I have tested with java 11 and java 17.

[jmeter.log](https://github.com/sngular/kloadgen/files/12396645/jmeter.log)

**Comment by jemacineiras** [comment]
Hi @xck1007,
at the moment our latest support version for JMeter is the 5.5. I´m taking a look about 5.6.2, looks there are some dependencies we need to change. Hope I will get a fix soon.

**Reply by xck1007** [reply]
I would be very happy about it
